utils/rpms/daos.spec

```diff
@@ -170,25 +170,20 @@ Requires: %{name}%{?_isa} = %{version}-%{release}
 Requires: mercury >= %{mercury_version}
 %if (0%{?rhel} >= 8)
 Requires: fuse3 >= 3
-Requires: capstone >= 4.0.1
 %else
 Requires: fuse3 >= 3.4.2
-Requires: capstone >= 4.0.1
 %endif
 Requires: libfabric >= %{libfabric_version}, libfabric < %{libfabric_max_version}
 %if (0%{?suse_version} >= 1500)
 Requires: libfabric1 >= %{libfabric_version}, libfabric1 < %{libfabric_max_version}
 Requires: libfuse3-3 >= 3.4.2
-Requires: libcapstone-devel >= 4.0.1
 %else
 # because our repo has a deprecated fuse-3.x RPM, make sure we don't
 # get it when fuse3 Requires: /etc/fuse.conf
 %if (0%{?rhel} >= 8)
 Requires: fuse3 >= 3
-Requires: capstone >= 4.0.1
 %else
 Requires: fuse < 3, fuse3-libs >= 3.4.2
-Requires: capstone >= 4.0.1
 %endif
```

Comment:
Looks like we still have some duplication here between these lines and https://github.com/daos-stack/daos/blob/41534deacee53adce1e2d095e24e5418adacc2dd/utils/rpms/daos.spec#L171-L175.

While it appears that this PR is not introducing the duplication, since we have discovered it here, it would be good to resolve it before it falls between the cracks again.

Reply:
Removed as you suggested. Thank you!